### PR TITLE
Feature/ghi 352 transaction fee components

### PR DESCRIPTION
**Pull request by sche**
transaction fee component

transaction_fee_view

### CODE REVIEW

File: SafeUIKit/SafeUIKit/Components/TransactionFeeView/TransactionFeeView.swift [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    }
+
+    private func pinWrapperStackView() {
+        wrapperStackView.axis = .vertical
+        wrapperStackView.spacing = 4
+        wrapperStackView.accessibilityIdentifier = "wrapperStackView"
+        wrapperStackView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(wrapperStackView)
+        NSLayoutConstraint.activate([
+            wrapperStackView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 16),
```
Comment (DmitryBespalov) [inline SafeUIKit/SafeUIKit/Components/TransactionFeeView/TransactionFeeView.swift:48]: refactoring proposal: extract the `16` and `20` into local variables (horizontal margin/padding, and vertical margin/padding)

File: SafeUIKit/SafeUIKit/Components/TransactionFeeView/TransactionFeeView.swift [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            addTransactionFee(transactionFee, isSecondaryView: isSecondaryView)
+        } else if !isSecondaryView {
+            wrapperStackView.spacing = 8
+        }
+        addResultingBalance(resultingBalance, isSecondaryView: isSecondaryView)
+    }
+
+    private func addCurrentBalance(_ currentBalance: TokenData) {
+        let currentBalanceStackView = balanceStackView(text: currentBalanceLabel(currentBalance),
+                                                       balance: balanceLabel(currentBalance,
```
Comment (DmitryBespalov) [inline SafeUIKit/SafeUIKit/Components/TransactionFeeView/TransactionFeeView.swift:70]: thank you for splitting the lines after commas 🙇

File: SafeUIKit/SafeUIKit/Components/TransactionFeeView/TransactionFeeView.swift [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                                                                             bold: true,
+                                                                             displayedDecimals: displayedDecimals))
+        wrapperStackView.addArrangedSubview(currentBalanceStackView)
+    }
+
+    private func addTransactionFee(_ transactionFee: TokenData, isSecondaryView: Bool) {
+        let transactionFeeStackView =
+            balanceStackView(text: transactionFeeLabel(transactionFee,
```
Comment (DmitryBespalov) [inline SafeUIKit/SafeUIKit/Components/TransactionFeeView/TransactionFeeView.swift:78]: 👍

**Review by DmitryBespalov — APPROVED** [review]

File: SafeUIKit/SafeUIKit/Components/TransactionFeeView/TransactionFeeView.swift (hunk 1 above)
Comment (sche) [reply SafeUIKit/SafeUIKit/Components/TransactionFeeView/TransactionFeeView.swift:48]: will do in next pr

**Pull request merged**